packages/lexical/src/nodes/LexicalParagraphNode.ts

```diff
@@ -38,8 +38,8 @@ import {$isTextNode, TextFormatType} from './LexicalTextNode';
 
 export type SerializedParagraphNode = Spread<
   {
-    textFormat: number;
-    textStyle: string;
+    textFormat?: number;
+    textStyle?: string;
```

Comment:
ParagraphNode shouldn't change in this way, it's not backwards compatible since these are existing and required, we would be generating JSON that the previous version may not be able to parse correctly.

```suggestion
    textFormat: number;
    textStyle: string;
```